src/shim/actors/miner/state.rs

```diff
@@ -114,4 +118,33 @@ impl MinerStateExt for State {
             }
         }
     }
+
+    fn find_sector<BS: Blockstore>(
```

Comment:
I would expect this method to be a part of `fil-actor-states` just like we do it for other actors. Then we don't need all of those `from...*` method in `forest` either.

Reply:
Here's a PR for inspiration: https://github.com/ChainSafe/fil-actor-states/pull/299

Basically we could do the same with `find_sector` and then just call that in `Forest` rpc code.

Reply:
I'd prefer getting rid of the process of updating and releasing `fil-actor-states` crates during the RPC development and we can backport all missing functions all at once after all RPC work is done. What do you think?

Reply:
`find_sector` added in https://github.com/ChainSafe/fil-actor-states/pull/303

> I'd rather go through an ordeal of having to reference a working branch in fil-actor-states 

Referencing a working branch without publishing would block `cargo publish --dry-run` for forest.

> introducing these helpers and then moving them to another repo.

I think the ordeal is because with the current development process of `fil_actor_interface`, there's no way to make the crate carefully designed, when the requirement of what are needed and what are not needed is unclear, and the rest of the crates in the repo are mostly copy-pasted code. And I really doubt the usefulness of `fil_actor_interface` after spending so much maintenance effort when it's not (and not likely possible to be ) a carefully design library.